server/server.go

```diff
@@ -520,18 +520,46 @@ func (s *Server) wipeIfAppropriate(ctx context.Context, imagesToDelete []string)
 	if !s.config.InternalWipe {
 		return
 	}
+	// Check if our version file is out of date.
+	// If so, we rebooted, and we should wipe containers.
+	shouldWipeContainers, err := version.ShouldCrioWipe(s.config.VersionFile)
```

Comment:
Does this also cover a reboot without a version change?

Reply:
yeah if the file isn't present it wipes